Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -10,17 +10,17 @@ This guide assumes a Rook cluster as explained in the [Quickstart](../../Getting
 
 ## Configure an Object Store
 
-Rook has the ability to either deploy an object store in Kubernetes or to connect to an external RGW service.
-Most commonly, the object store will be configured locally by Rook.
-Alternatively, if you have an existing Ceph cluster with Rados Gateways, see the
-[external section](#connect-to-an-external-object-store) to consume it from Rook.
+Rook can configure the Ceph Object Store for several different scenarios. See each linked section for the configuration details.
+1. Create a [local object store](#create-a-local-object-store) with dedicated Ceph pools. This option is recommended if a single object store is required, and is the simplest to get started.
+2. Create [one or more object stores with shared Ceph pools](#create-local-object-stores-with-shared-pools). This option is recommended when multiple object stores are required.
+3. Connect to an [RGW service in an external Ceph cluster](#connect-to-an-external-object-store), rather than create a local object store.
 
 ### Create a Local Object Store
 
 The below sample will create a `CephObjectStore` that starts the RGW service in the cluster with an S3 API.
 
 !!! note
-    This sample requires *at least 3 bluestore OSDs*, with each OSD located on a *different node*.
+    This sample requires *at least 3 OSDs*, with each OSD located on a *different node*.
```

Comment:
It seems like the asterisk after "OSDs*" might be accidental. Should we remove it?

Reply:
In the docs looks like it's rendered as italics from the beginning asterisk to the last one, so it looks valid to me.